### PR TITLE
docs: Fix grammar issues as Grammarly suggested

### PR DESCRIPTION
**Pull request by iosmanthus**
Signed-off-by: Iosmanthus Teng <myosmanthustree@gmail.com>

## What have you changed? (mandatory)
Fix grammar issues as [Grammarly](https://app.grammarly.com) suggested. Every time I try to open a pull request, Grammarly complains the grammar issues of the pull request template. 

## What are the type of changes? (mandatory)

The currently defined types are listed below, please pick one of the types for this PR by removing the others:
- Misc (other changes)

## How has this PR been tested? (mandatory)
No need to test.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -13,12 +13,12 @@ Please explain **IN DETAIL** what the changes are in this PR and why they are ne
...
-## What are the type of the changes? (mandatory)
+## What are the type of changes? (mandatory)
 
 The currently defined types are listed below, please pick one of the types for this PR by removing the others:
-- New feature (change which adds functionality)
-- Improvement (change which is an improvement to an existing feature)
-- Bug fix (change which fixes an issue)
+- New feature (a change which adds functionality)
+- Improvement (a change which is an improvement to an existing feature)
+- Bugfix (a change which fixes an issue)
```
Comment (mapleFU) [inline .github/pull_request_template.md]: Maybe `changes`, like `What are the type of changes?`

**Comment by siddontang** [comment]
CI failed

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -13,12 +13,12 @@ Please explain **IN DETAIL** what the changes are in this PR and why they are ne
...
 - Do not assume reviewers understand the original issue
 
-## What are the type of the changes? (mandatory)
+## What are the type of changes? (mandatory)
 
 The currently defined types are listed below, please pick one of the types for this PR by removing the others:
-- New feature (change which adds functionality)
-- Improvement (change which is an improvement to an existing feature)
-- Bug fix (change which fixes an issue)
+- New feature (a change which adds functionality)
```
Comment (breeswish) [inline .github/pull_request_template.md]: How many changes? One or many? In title it is "changes" but here it is "a change".

**Comment by siddontang** [comment]
CI failed

**Comment by breeswish** [comment]
How is the status of the comment? @iosmanthus @dcalvin @CaitinChen

File: .github/pull_request_template.md (hunk 2 above)
Comment (dcalvin) [reply .github/pull_request_template.md]: Maybe change the title to "What is the type of the change?"

**Pull request merged**